packages/block-editor/src/components/list-view/appender.js

```diff
@@ -88,6 +121,11 @@ export const Appender = forwardRef(
 							setInsertedBlock( maybeInsertedBlock );
 						}
 					} }
+					orderInitialBlockItems={
```

Comment:
We should remove this since we're going to do it this way instead: https://github.com/WordPress/gutenberg/pull/50510